dynawo/sources/Models/CPP/Controls/Shunts/ModelCentralizedShuntsSectionControl/DYNModelCentralizedShuntsSectionControl.cpp

```diff
@@ -179,26 +175,26 @@ namespace DYN {
     double UMonitoredPu = yLocal_[0];
     double minValue;
     double maxValue;
-    if (!isSelf_) {
+    if (isSelf_) {
       for (int s = 0; s < nbShunts_; ++s) {
-        minValue = zLocal_[s] - DeadbandsUPu_[s];
-        maxValue = zLocal_[s] + DeadbandsUPu_[s];
+        minValue = zLocal_[0] - deadBandsUPu_[s];
```

Comment:
Could you please create an enum or a constant for this 0 index? something like URefPuIdx

Reply:
done!